src/main/resources/db/migration/V1__Hibernate_Sequence.sql

```diff
@@ -0,0 +1,2 @@
+-- The default sequence used to automatically populate primary keys.
+CREATE SEQUENCE hibernate_sequence;
```

Comment:
Why this is required?

When dealing with Postgres, in the case of a numeric PK, you can set the type of the column as serial, and Postgres will create a sequence and use it for you. You can then set the id to be `@GeneratedValue(strategy=GenerationType.IDENTITY)` and the type as `Long` and it should be more than enough.